src/Machine.Specifications.Runner.VisualStudio/Machine.Specifications.Runner.VisualStudio.csproj

```diff
@@ -40,19 +34,24 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);NetCorePackageItems;NetFrameworkPackageItems</TargetsForTfmSpecificContentInPackage>
   </PropertyGroup>
 
+  <Target Name="PreventTestPlatformObjectModelCopyLocal" AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="'%(Filename)' == 'Microsoft.VisualStudio.TestPlatform.ObjectModel'" />
+    </ItemGroup>
+  </Target>
+
   <Target Name="NetCorePackageItems" Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
     <ItemGroup>
-      <TfmSpecificPackageFile Include="*.props" PackagePath="build\$(TargetFramework)" />
-      <TfmSpecificPackageFile Include="$(OutputPath)\*.dll" PackagePath="build\$(TargetFramework)" />
+      <TfmSpecificPackageFile Include="*.props" PackagePath="lib\$(TargetFramework)" />
+      <TfmSpecificPackageFile Include="$(OutputPath)\Machine.*.dll" PackagePath="lib\$(TargetFramework)" />
     </ItemGroup>
   </Target>
 
-  <Target Name="NetFrameworkPackageItems" Condition="'$(TargetFramework)'=='net452'">
+  <Target Name="NetFrameworkPackageItems" Condition="'$(TargetFramework)' != 'netcoreapp2.1'">
     <ItemGroup>
-      <TfmSpecificPackageFile Include="*.props" PackagePath="build\$(TargetFramework)" />
-      <TfmSpecificPackageFile Include="$(OutputPath)\Machine.Specifications.dll" PackagePath="build\$(TargetFramework)\" />
-      <TfmSpecificPackageFile Include="$(OutputPath)\Machine.TestAdapter.dll" PackagePath="build\$(TargetFramework)" />
-      <TfmSpecificPackageFile Include="$(OutputPath)\Mono.*.dll" PackagePath="build\$(TargetFramework)" />
+      <TfmSpecificPackageFile Include="*.props" PackagePath="lib\$(TargetFramework)" />
+      <TfmSpecificPackageFile Include="$(OutputPath)\Machine.*.dll" PackagePath="lib\$(TargetFramework)" />
+      <TfmSpecificPackageFile Include="$(OutputPath)\Mono.*.dll" PackagePath="lib\$(TargetFramework)" />
```

Comment:
Revert all this please, we want to be specific about what goes in the package. Also you've changed the path, which is the wrong path, it should be `build`.